### PR TITLE
update some example imagepullpolicy to IfNotPresent

### PR DESCRIPTION
**Pull request by my-git9**
**Please provide a description of this PR:**

Update some example imagepullpolicy to IfNotPresent

### CODE REVIEW

**Comment by craigbox** [comment]
I went through the PRs that committed each and didn't see any indication they need to be set to `Always`.

**Review by craigbox — APPROVED** [review]

**Comment by ericvn** [comment]
Note that setting them to IfNotPresent will not force a reload of newer images if they exist. I believe in the past, we always wanted to force a user to use the latest and greatest images.

**Review by ericvn — CHANGES_REQUESTED** [review]
More discussion needed if we want to make sure the user is using the latest images.

**Comment by craigbox** [comment]
Most of the existing samples are `imagePullPolicy: ifNotPresent` already. Happy to have the discussion but I saw this PR as being just bringing a couple of outliers into line.

(I will say that when people benchmark things like Bookinfo, we don't want the container re-pulled every time.)

File: samples/httpbin/sample-client/fortio-deploy.yaml [hunk 1]
```diff
@@ -41,7 +41,7 @@ spec:
       containers:
       - name: fortio
         image: fortio/fortio:latest_release
-        imagePullPolicy: Always
+        imagePullPolicy: IfNotPresent
```
Comment (zirain) [inline samples/httpbin/sample-client/fortio-deploy.yaml:44]: IMO, should keep this as `Always` because it's not static.

**Pull request merged**